net/mosquitto/Portfile

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8; mode: tcl; indent-tabs-mode: nil -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           cmake 1.1
+
+name                mosquitto
+version             1.4.14
+
+categories          net devel
+platforms           darwin
+license             EPL-1.0 and EDL-1.0
```

Comment:
The license should be written like this:
`license           {EPL-1.0 EDL-1.0} `